src/components/ShareStructure/ShareStructure.vue

```diff
@@ -921,11 +919,6 @@ export default class ShareStructure extends Mixins(ShareMixin) {
     return new Array(50).fill(null).map(() => new Array(50).fill(null))
   }
 
-  @Watch('businessSnapshot')
-  private updateSnapshot (): void {
-    this.originalSnapshot = cloneDeep(this.businessSnapshot)
```

Comment:
No need to watch the prop and copy to a local variable. Instead, the script just uses the prop directly.

Reply:
Ok 👍